From cf89b2152b504b1a645382278236174b9bd48a99 Mon Sep 17 00:00:00 2001
From: Geod24 <pro.mathias.lang@gmail.com>
Date: Fri, 11 Jun 2021 17:18:41 +0900
Subject: [PATCH 1/2] Unreserve CppRuntime identifiers

This way we can define them from user code,
but one has to know what they are doing,
otherwise we'll end up with a mismatch between the version
and __traits(getTargetInfos).
---
 dmd/cond.d | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/dmd/cond.d b/dmd/cond.d
index 672768e0..2f8c88b0 100644
--- a/dmd/cond.d
+++ b/dmd/cond.d
@@ -646,11 +646,6 @@ extern (C++) final class VersionCondition : DVCondition
             case "AVR":
             case "BigEndian":
             case "BSD":
-            case "CppRuntime_Clang":
-            case "CppRuntime_DigitalMars":
-            case "CppRuntime_Gcc":
-            case "CppRuntime_Microsoft":
-            case "CppRuntime_Sun":
             case "CRuntime_Bionic":
             case "CRuntime_DigitalMars":
             case "CRuntime_Glibc":
-- 
2.30.1 (Apple Git-130)

